docs/2.8.0/docs/daml-finance/concepts/lifecycling.rst

```diff
@@ -94,6 +96,9 @@ can be used to create new instrument versions and associated lifecycle events.
    :alt: The issuer creates a new ACME v2 instrument. They also create a distribution
          event by declaring a dividend on the ACME v1 instrument.
 
+Note that there is only one instrument (the v1) paying a dividend, so there will be one distribution
```

Comment:
The feedback from Shaul is to remind the reader that only one instrument exists, rather than the implication one instrument => one event.

I would suggest something along the lines of

```suggestion
The lifecycle event will now be used to evolve the single instrument contract for the v1 (dividend paying) instrument.
```

wdyt?

Reply:
Not sure I understand the suggested formulation. In order to remove the implication one instrument => one event, what about something like:
_Note that there is only one instrument (the v1) paying a dividend. The lifecycle event will now be used to evolve the v1 contract into a v2, that does not pay a dividend._

Reply:
Aha, I think we have a different understanding of the feedback item at hand. 

> In creating-the-event I recommend reminding the reader that there is only one instrument, so there will be one distribution event

I read it as
- remind the reader that there is only one instrument contract that needs to be processed

I believe your interpretation is
- remind the reader that the event will be used to evolve `V1` and not `V1` and `V2`. 

I like yours better :)

To be fair, I don't know why we do not encode the creation of the `V2` instrument directly in the distribution rule, it would make things simpler.

Reply:
> To be fair, I don't know why we do not encode the creation of the `V2` instrument directly in the distribution rule, it would make things simpler.

Do you want to open a line in the UX sheet for this?

Reply:
I have reformulated it now.